ldi-core/ldes-client/tree-node-relations-fetcher/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>ldes-client</artifactId>
+        <groupId>be.vlaanderen.informatievlaanderen.ldes.client</groupId>
+        <version>1.15.0-SNAPSHOT</version>
+    </parent>
+    <dependencies>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.client</groupId>
+            <artifactId>starting-node</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>tree-node-relations-fetcher</artifactId>
```

Comment:
Mvn structure: first parent, then (optionally) groupId, then artifactId, then dependencies/build.
Model version if possible should be ommitted